.github/workflows/build.yml

```diff
@@ -81,5 +81,5 @@ jobs:
       - name: Lint
         run: |
           pip install black==22.3.0 isort
-          black . --check -v
-          isort . --check -v
+          isort --profile black .
```

Comment:
Not gonna lie, I'm not sure why this fix worked but previously we were having isort fail despite having it work fine when running locally (I confirmed same versions were being used).

Ref: https://pycqa.github.io/isort/docs/configuration/black_compatibility.html

Reply:
I think we'd want to at least remove --diff and add -v back to both commands -- those are somewhat important (first ensures we can't get into a bad state even if we merge a PR with failing CI, second allows us to see the details of where the failure is in the logs).

And, if it were to depend on the order the commands were run (which you've switched here), I think we'd want to know why.

Reply:
Ah, I bet it has to do with `--diff`: the "push" actions are succeeding while the "pull request" actions are failing.

Reply:
fwiw the framework code all uses 
```
black . --check --diff
isort . --check --verbose --only-modified --diff
```

But that fails as well